### DISCUSSION TITLE
BILATERAL_COMBINATIONS don't seem to work?

### DISCUSSION THREAD

**Discussion by Exegetech** [manna-harbour/miryoku#277] [Q&A] [answered]
I am trying the Miryoku layout, using QWERTY in Iris (Keebio) Rev6. I want to enable BILATERAL_COMBINATIONS because I often accidentally close a browser window when I roll over typing aw (it becomes Cmd + W on Mac).

I already put this code

```h
#define BILATERAL_COMBINATIONS
```

in the `users/manna-harbour_miryoku/config.h` but it doesn't seem to work? Sorry if this is a really dumb question.

**Comment by manna-harbour** [comment, marked as answer]
Bilateral Combinations is a patch to QMK, and so isn't included with Miryoku QMK.

https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku#bilateral-combinations links to https://github.com/manna-harbour/qmk_firmware/issues/29, which gives the branch that needs to merged. The branch isn't maintained, so may be difficult to merge.

If you don't want to deal with merge conflicts, consider using https://getreuer.info/posts/keyboards/achordion/ instead.

Alternatively, you could increase tapping term by editing https://github.com/manna-harbour/miryoku_qmk/blob/df1690ad257e7b0d1bf9450258f7baebfee7be9a/users/manna-harbour_miryoku/config.h#L12, or try not to linger when rolling.
